modules/picard/collectwgsmetrics/main.nf

```diff
@@ -32,9 +32,9 @@ process PICARD_COLLECTWGSMETRICS {
         -Xmx${avail_mem}g \\
         CollectWgsMetrics \\
         $args \\
-        INPUT=$bam \\
-        OUTPUT=${prefix}.CollectWgsMetrics.coverage_metrics \\
-        REFERENCE_SEQUENCE=$fasta
+        -INPUT $bam \\
```

Comment:
guess will have to update the other picard modules as well in one picard upgrade